src/main/java/com/Elessar/database/MetricMonitor.java

```diff
@@ -0,0 +1,56 @@
+package com.Elessar.database;
+
+
+import com.Elessar.app.server.RegisterHandler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Created by Hans on 1/26/19.
+ */
+public class MetricMonitor {
+    private static final Logger logger = LogManager.getLogger(MetricMonitor.class);
+    private int count;
+    private long totalTime;
+    private long start;
+    public MetricMonitor() {
+        this.count = 0;
+        this.totalTime = 0;
+        this.start = 0;
+    }
+
+    /**
+     * Get average time
+     * @return average time per op in milliseconds
+     */
+    public double getAvgTime() {
+        if (count == 0) {
+            return 0;
+        }
+        return (double) totalTime / count;
+    }
+
+    public void timerStart() {
+        start = System.currentTimeMillis();
+    }
+
+    public double timerEnd() {
+        if (start == 0) {
```

Comment:
this means a developer made incorrect use of the library, we should either assert or throw exception here.